tools/iptb-plugins/filecoin/local/localfilecoin.go

```diff
@@ -38,6 +38,9 @@ var errTimeout = errors.New("timeout")
 // DefaultFilecoinBinary is the name or full path of the binary that will be used
 var DefaultFilecoinBinary = "go-filecoin"
 
+// DefaultSectorsPath is the name of the sector path relative to the repo path
+var DefaultSectorsPath = "sectors"
```

Comment:
Putting the sectors inside the repo differs from the way every other use case will lay things out, which could cause problems later.